docs/patterns/single-new-eks-observability-accelerators/single-new-eks-java-opensource-observability.md

```diff
@@ -0,0 +1,124 @@
+# Single New EKS Cluster Open Source Observability Accelerator - monitoring Java applications
```

Comment:
Where is the pattern code for this doc?

Reply:
small code changes to support new java flag and cdk file config. main code changes are in otel collector config (see quickstart PR).